Sage-Bionetworks/Bridge-Exporter
# migrate to new Bridge REST Client

This migrates to the new Bridge REST Client, which ensures that we're using the most up-to-date APIs and library versions.

Testing done:
* mvn verify (unit tests, findbugs, jacoco test coverage)
* manual tests

Manual Testing done:
* export, using schema upload-v2-manual-test to test a variety of fields
* verified recordExportStatus
* upload autocomplete
* refresh sessions (both export and upload autocomplete)
